docker-compose/technical/init-geo-data.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+
+function curl_()
+{
+  curl -f -s -o /dev/null -H "Content-Type: application/json" "$@"
+}
+
+function init_geo_data()
+{
+  FILE_SUBSTATIONS=/init-data/geo_data_substations.json
+  FILE_LINES=/init-data/geo_data_lines.json
+
+  ([ ! -f "$FILE_SUBSTATIONS" ] || curl_ -d@$FILE_SUBSTATIONS http://geo-data-server/v1/substations) \
```

Comment:
We could even download it from opendata website here no ?

Reply:
See the readme